Parser/Token/TerminatorToken.php

```diff
@@ -45,9 +45,11 @@ protected function processStart(Context $context, Language $language, Result $re
 
     protected function processEnd(Context $context, Language $language, Result $result, TokenIterator $tokens)
     {
-        foreach(array_filter($context->stack, function ($name) {
-            return in_array($name, $this->closes);
-        }) as $hash => $name) {
+        foreach (
+            array_filter($context->stack, function ($name) {
```

Comment:
If we are changing it maybe it 'd be better to create some kind of temporary variable like `$closing`?